net/mwan3/files/usr/libexec/rpcd/mwan3

```diff
@@ -77,70 +77,70 @@ get_mwan3_status() {
 	local online=0
 	local offline=0
 	local up="0"
-	local enabled device time_p time_n time_u time_d status track_status
```

Comment:
Could you also please move this two separate commits?
- One for remove unnecessary network_get_device_call from rpcd
- And another for fixup indenting on status call

Then this is enforceable.